### PR TITLE
Switch to c2cwsgiutils

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

**Comment by pvalsecc** [comment]
I'll do the health check integration in another PR

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 BUILD_DIR ?= /build
-TEMPLATE_EXCLUDE += c2cgeoportal/scaffolds tests/testegg c2cgeoportal/templates ngeo
+TEMPLATE_EXCLUDE += .build .venv c2cgeoportal/scaffolds tests/testegg c2cgeoportal/templates ngeo
```
Comment (sbrunner) [inline Makefile]: => TEMPLATE_EXCLUDE += c2cgeoportal/scaffolds tests/testegg c2cgeoportal/templates

File: c2cgeoportal/pyramid_.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+# -*- coding: utf-8 -*-
```
Comment (sbrunner) [inline c2cgeoportal/pyramid_.py]: Tow times ?

File: tests/functional/__init__.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+# -*- coding: utf-8 -*-
```
Comment (sbrunner) [inline tests/functional/__init__.py]: tow times ...

File: Makefile [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
 FIND_OPTS = $(foreach ELEM, $(TEMPLATE_EXCLUDE),-path ./$(ELEM) -prune -o) -type f
```
Comment (sbrunner) [inline Makefile]: FIND_OPTS = tests doc c2cgeoportal .tx  $(...

File: Makefile (hunk 1 above)
Comment (pvalsecc) [reply Makefile]: It was you who added it. For avoiding trouble when we have a .venv for the IDE.
Removing .build, though

Comment (sbrunner) [reply Makefile]: see FIND_OPTS comment to solve that :-)

Comment (pvalsecc) [reply Makefile]: With this modif, the tests are failing...

Comment (sbrunner) [reply Makefile]: and with 
TEMPLATE_EXCLUDE += scaffolds templates
FIND_OPTS = tests/functional doc c2cgeoportal .tx $(...
?

**Comment by sbrunner** [comment]
find .tx doc docker tests -type f -name *.mako

**Comment by sbrunner** [comment]
I did a new commit, and now it's working well :-), can you amend it and merge :-)

**Pull request merged**